src/components/Column/style.scss

```diff
@@ -1,165 +1,184 @@
 .column-container {
-  margin: 8px;
-  width: 19.15%;
+  margin-left: 15px;
+  margin-top: 15px;
+  width: 270px;
 
   .column-header-container {
-    background-color: rgb(255, 255, 255);
-    height: 110px;
-    margin-top: 4px;
+    background-color: rgb(228, 220, 255);
     font-size: 18px;
-    color: rgb(129,131,133);
-    font-family: sans-serif;
+    font-family: Exo;
     box-shadow: 1px 1px rgba(0,0,0,0.2);
-    border-radius: 4px;
+    border-top-left-radius: 4px;
+    border-top-right-radius: 4px;
+    padding-bottom: 15px;
 
+    &.no-card {
+      border-radius: 5px;  
+    }
+    
     .column-header {
+      font-family: Exo;
```

Comment:
Are we going to have different fonts in the project?
Let's set in the body. So we don't have to set it any more times.